dcc-portal-ui/app/scripts/common/js/common.js

```diff
@@ -379,4 +379,11 @@
     };
   });
 
+  // This is a workaroud required for Internationalization of 'Experimental&nbsp;Strategy'
+  module.filter('replace', function(){
+    return function(string, oldValue, newValue){
+      return string.replace(oldValue, newValue);
```

Comment:
very close!👍 
unfortunately in js this will only replace the first occurrence
need a regex, see

```js
function replace (input, pattern, replacement) {
  return input.replace(new RegExp(pattern, 'g'), replacement)
}
```